### DISCUSSION TITLE
Freshwater timeseries progress and questions

### DISCUSSION THREAD

**Discussion by mankoff** [NASA-GISS/freshwater-forcing-workshop#22] [Timeseries development]
Hello,

I've made some progress on the freshwater time series. This is an update, and request for review / input / thoughts on what changes are needed to make this more useful for the models.

I've been working in *this* repository (the freshwater-forcing-workshop at https://github.com/NASA-GISS/freshwater-forcing-workshop) and the README.org file you see there has been updated as I work. It now has sections for the following areas, and [X] marks work I've completed:

+ [-] Antarctic
  + [X] Iceberg Days workbook: https://github.com/NASA-GISS/freshwater-forcing-workshop/blob/main/iceberg_days.org
  + [ ] Iceberg melt
  + [ ] Ice shelf melt
  + [ ] Surface runoff
  + [ ] Basal (grounded) runoff
+ [-] Greenland
  + [ ] Basal runoff
  + [X] Surface runoff workbook: https://github.com/NASA-GISS/freshwater-forcing-workshop/blob/main/greenland_runoff.org
  + [X] Glacier termini workbook: https://github.com/NASA-GISS/freshwater-forcing-workshop/blob/main/greenland_discharge.org
    + [ ] Calving (solid)
    + [ ] Submarine melt (liquid)

This is preliminary work, and has the following limitations
+ Iceberg days: Only where icebergs reside, no melt
+ Surface runoff:
  + Only back to 1950
  + Includes rain-on-ice (for ice runoff) and rain-on-land (for land runoff). Is this a problem?
+ Glacier termini
  + Not sure how to split between calving vs submarine melt.
  + Only back to 1980
+ Various other issues
  + Not all data files are identical/homogeneous
  + Other issues I've thought about for each product are discussed in more detail in each workbook
  + Other things likely to come up through discussion with others vs solo work

To do:    
+ Glacier termini discharge back to 1850 is done elsewhere (by me and @jlbamber) and I can bring that work in, but at least my version is GIS-summed, not broken down by ROI.
+ Surface runoff can be correlated with glacier termini discharge, but errors grow...

**Comment by mankoff** [comment]
### Runoff includes rain

Current Greenlandic runoff products (Mankoff 2020, Bamber 2018 about-to-be-updated) define runoff as

> R = ME + RA − RT − RF

with ME = melt, RA = rain, RT = retention, and RF = refreeze. That is, runoff includes rain.

How problematic is this? Is it just an X % error, where `X = RA/R*100` and we approximate it, document it, and scale R by that amount, assuming it doesn't change (much) with time? Can we leave it as-is, document it, and recommend the GCMs use their own rainfall estimates to adjust? Or should these products be rebuilt w/o `RA`?

**Comment by gavinny** [comment]
But all of these terms calculated by the models anyway, no?

--

Gavin A. Schmidt
Director, Goddard Institute for Space Studies
2880 Broadway, New York NY 10025
Cell: 212 749 0006
Email: ***@***.******@***.***>
URL: https://www.giss.nasa.gov/staff/gschmidt



From: Ken Mankoff ***@***.***>
Date: Tuesday, May 14, 2024 at 10:34 AM
To: NASA-GISS/freshwater-forcing-workshop ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [EXTERNAL] [BULK] Re: [NASA-GISS/freshwater-forcing-workshop] Freshwater timeseries progress and questions (Discussion #22)
CAUTION: This email originated from outside of NASA.  Please take care when clicking links or opening attachments.  Use the "Report Message" button to report suspicious messages to the NASA SOC.


Runoff includes rain

Current Greenlandic runoff products (Mankoff 2020, Bamber 2018 about-to-be-updated) define runoff as

R = ME + RA − RT − RF

with ME = melt, RA = rain, RT = retention, and RF = refreeze. That is, runoff includes rain.

How problematic is this? Is it just an X % error, where X = RA/R*100 and we approximate it, document it, and scale R by that amount, assuming it doesn't change (much) with time? Can we leave it as-is, document it, and recommend the GCMs use their own rainfall estimates to adjust? Or should these products be rebuilt w/o RA?

**Reply by mankoff** [reply]
The GISS snow model calculates these terms, but large volumes of freshwater come from melted ice, not only seasonal snow. In our GCM, that's treated as icebergs, which means that freshwater enters the oceans in the wrong location (x,y,z) and phase (energy).

This is likely to be different in other models.
